.travis.yml

```diff
@@ -13,4 +18,6 @@ before_script:
         else
             PYTEST_ADDOPTS="-m 'not optional'"
         fi
-script: make PYTEST_ADDOPTS="${PYTEST_ADDOPTS}" test-docker
+script:
+    - docker version
```

Comment:
On the other hand, the version is visible in apt-get output, so you can go ahead and just remove it.